march_simulation/scripts/upload_joint_names

```diff
@@ -6,7 +6,7 @@ import rospy
 
 def main():
     rospy.init_node("upload_joint_names")
-    robot = urdf.Robot.from_parameter_server()
+    robot = urdf.Robot.from_parameter_server('/robot_description')
```

Comment:
Isn't this the default anyway?

Reply:
Almost, the default is `'robot_description'`, which resolves to the current namespace. So when the node is in namespace `march` it will resolve to `/march/robot_description`. With this change it will always resolve to `/robot_description`.

Reply:
Interesting,

is it a deliberate choice to not upload the /robot_description under the march namespace?

Reply:
I wasn't sure how that change would affect the other nodes. And it would change whenever the node was launched in a different namespace, so I kept it at the global namespace